### DISCUSSION TITLE
Can you display the map in bird view?

### DISCUSSION THREAD

**Discussion by tjahnn** [fleaflet/flutter_map#1246] [Q&A] [answered]
If you look at the Google Maps API, you give the tilt angle to implement the bird view.

Is there such an option in flutter map ?

![image](https://user-images.githubusercontent.com/29188211/170516205-a6c6b2a4-bfdb-4c9e-bbe7-6f90f7ca5fe5.png)

**Comment by ibrierley** [comment, marked as answer]
There isn't currently such an option. It is likely feasible to implement with some hackery (I've done similar a long while back), but it tends to break things like dragging/pinchzooming if needed and would probably need a fair bit of work to get something as a proper solution that could go into production.
